config/generic-mellow-fly-d5.cfg

```diff
@@ -0,0 +1,131 @@
+#Mellow Fly D5 generic printer.cfg
+#Check https://www.klipper3d.org/Config_Reference.html before use.
+#Pruduct page on https://mellow.klipper.cn/en/docs/category/fly-d5
+#STM32F072, 8KiB bootloader, 8 Mhz crystal, USB or USB to CAN adapter(depends on if you are using a canbus toolhead board)
+#See more about firmwares on https://mellow.klipper.cn/en/docs/ProductDoc/MainBoard/fly-d/fly-d5/flash/
+#This document is mainly used for pin references, change other parameters before actual use.
```

Comment:
Please follow the method / layout as with the other example configs. As the availability of the links cannot be guaranteed, at least the relevant pins for USB and/or CAN should be provided.